commands/_mongoFunctions.py

```diff
@@ -28,10 +32,57 @@ def init():
     guild_list = list(Guilds.find({}))
 
     for guild in guild_list:
-        for key, value in guild.items():
-            if key == 'guild_id':
-                coll = GuildInformation["a" + str(value) + ".PendingVerificationUsers"]
-                coll.delete_many({})
+        Guild_Cache[str(guild['guild_id'])] = {}
+        Guild_Cache[str(guild['guild_id'])]['settings'] = guild
+
+        pending_verification_coll = GuildInformation["a" + str(guild['guild_id']) + ".PendingVerificationUsers"]
+        pending_verification_coll.delete_many({})
+
+    new_loop = asyncio.new_event_loop()
+    t = threading.Thread(target = start_loop, args = (new_loop,))
+    t.start()
+
+    asyncio.run_coroutine_threadsafe(update_guilds(client), new_loop)
+
+
+def start_loop(loop):
+    asyncio.set_event_loop(loop)
+    loop.run_forever()
```

Comment:
Can you add a comment to the purpose of this function so that we aren't lost 2 months down the line lmao